janus_core/cli.py

```diff
@@ -210,16 +237,74 @@ def singlepoint(
     if out_file:
         write_kwargs["filename"] = out_file
 
-    s_point = SinglePoint(
-        struct_path=struct_path,
-        architecture=architecture,
-        device=device,
-        read_kwargs=read_kwargs,
-        calc_kwargs=calc_kwargs,
-        log_kwargs={"filename": log_file, "filemode": "w"},
-    )
+    singlepoint_kwargs = {
+        "struct_path": struct_path,
+        "architecture": architecture,
+        "device": device,
+        "read_kwargs": read_kwargs,
+        "calc_kwargs": calc_kwargs,
+        "log_kwargs": {"filename": log_file, "filemode": "w", "force": True},
+    }
+
+    # Initialise singlepoint structure and calculator
+    s_point = SinglePoint(**singlepoint_kwargs)
+
+    # Store inputs for yaml summary
+    inputs = {}
+    for key, value in singlepoint_kwargs.items():
+        inputs[key] = value
```

Comment:
```suggestion
    inputs = single_point_kwargs.copy()
```
is best, but you can also (but here shouldn't) use dict unpacking:
```suggestion
    inputs = {**singlepoint_kwargs}
```
or with a dict comprehension
```suggestion
    inputs = {key: value for key, value in singlepoint_kwargs.items()}